lib/EntryManager/EntryForm.js

```diff
@@ -148,7 +148,7 @@ class EntryForm extends React.Component {
 
             {selectedEntry &&
               <ConfirmationModal
-                id={`delete${this.props.entryLabel.replace(/[^a-zA-Z0-9]/g, '').toLowerCase()}-confirmation`}
+                id={`delete${this.props.nameKey.replace(/[^a-zA-Z0-9]/g, '').toLowerCase()}-confirmation`}
```

Comment:
Heads up @zburke this id was reliant on a translated string, so I changed it to a stable value. Likely will cause an integration test failure.